sample-code/javascript-wd/test/helpers/caps.js

```diff
@@ -0,0 +1,23 @@
+import path from 'path';
+
+const iosCaps = {
+  platformName: 'iOS',
+  automationName: 'XCUITest',
+  deviceName: 'iPhone 6s' || process.env.IOS_DEVICE_NAME,
```

Comment:
This will _always_ set `deviceName` to `'iPhone 6s'`, since that string is always truthy. These need to be switched around:
```js
deviceName: process.env.IOS_DEVICE_NAME ||  'iPhone 6s',
platformVersion: process.env.IOS_PLATFORM_VERSION || '11.2',
```